### DISCUSSION TITLE
3AA4DB07-2BED-4CB1-AA56-7819DC99AD41

### DISCUSSION THREAD

**Discussion by giscus** [celestiamobile/comments#29] [General]
# Barycenter

A fictional barycenter placed around the Sun with multiple planets each featuring a texture at some point used as a generic exoplanet texture.

https://celestia.mobi/addon?item=3AA4DB07-2BED-4CB1-AA56-7819DC99AD41&lang=en

**Comment by Dirgham2015** [comment]
OoOoOh

**Reply by zjx29rjhdh-pixel** [reply]
Yeah, VERY COOL !

**Comment by zjx29rjhdh-pixel** [comment]
VERY COOL! Two habitable planets (one of is a copy of Earth) , three binary gas giants and two binary gas giants binary with other two binary gas giants , a copy of Titan, a copy of Ganymede and not last, a brown dwarf binary with a gas giant (that I think that has a lot of mass)

**Reply by zjx29rjhdh-pixel** [reply]
Now , because of this add-on and other add-ons (of fictional planets), my default Solar System has 22 planets (in order of distance of Sun):Infare, Vulcan, Blue, Mercury, Venus, Earth&Seranna&Theia (they’re at the same distance from Sun), Exo Gas-Class 1 , Exo Gas-Class 2, Gas Giant, Jupiter like, Mars, Planet V, Phaeton, Saturn (I didn’t write Jupiter because I have it as a red dwarf star), Fifth Giant, Uranus , Neptune (I didn’t write Nibiru because it is a brown dwarf ), Planet Ten (Terminus), Planet X, Planet Nine, Tyche (disproven).

**Comment by LangLangFCS** [comment]
😎 Cool.It's very nice.
